pom.xml

```diff
@@ -1984,6 +1984,7 @@
             <excludedLink>https://paypal.com/</excludedLink>
             <!-- temporal suppress till plugin update link in his source -->
             <excludedLink>http://www.mojohaus.org/exec-maven-plugin</excludedLink>
+            <excludedLink>http://www.mojohaus.org/xml-maven-plugin</excludedLink>
```

Comment:
This one gives 301. I noticed similar links were present in `pom.xml` so I placed it next to them.
```
------------ grep of linkcheck.html--BEGIN
1a2
> <td><i><a class="externalLink" href="http://www.mojohaus.org/xml-maven-plugin/">http://www.mojohaus.org/xml-maven-plugin/</a>: 301 Moved Permanently</i></td></tr></table></td></tr>
------------ grep of linkcheck.html--END
Exit code:1

```

Reply:
Moving to a separate PR to merge faster.

Reply:
Resolved.